extension.json

```diff
@@ -64,6 +64,11 @@
 			"description": "Sets the external link target (_self for the current tab or _blank for a new tab). (string)",
 			"public": true,
 			"value": "_self"
+		},
+		"RottenLinksUserAgent": {
+			"description": "Overrides the user-agent to use for requests. Defaults to 'RottenLinks, MediaWiki extension (https://github.com/miraheze/RottenLinks), running on <Wiki base URL>'. (string)",
+			"public": true,
+			"value": null
```

Comment:
Can this default to an empty string since we are declaring the type as a string?